Lib/asyncio/streams.py

```diff
@@ -766,18 +391,23 @@ def get_extra_info(self, name, default=None):
             if exc is not None:
                 raise exc
         if self._transport.is_closing():
+            # Wait for protocol.connection_lost() call
+            # Raise connection closing error if any,
+            # ConnectionResetError otherwise
             # Yield to the event loop so connection_lost() may be
             # called.  Without this, _drain_helper() would return
             # immediately, and code that calls
             #     write(...); await drain()
             # in a loop would never call connection_lost(), so it
             # would not see an error when the socket is closed.
-            await tasks.sleep(0, loop=self._loop)
+            await sleep(0)
```

Comment:
Same as 3.9 PR.
```suggestion
             # Yield to the event loop so connection_lost() may be 
             # called.  Without this, _drain_helper() would return 
             # immediately, and code that calls 
             #     write(...); await drain() 
             # in a loop would never call connection_lost(), so it 
             # would not see an error when the socket is closed. 
             await tasks.sleep(0, loop=self._loop) 
```

Reply:
I've actually merged the comment, but the `await sleep(0)` line is correct.

Reply:
> but the await sleep(0) line is correct

Oh you're right, there's no reason to explicitly pass the loop argument there, that wasn't intentional.